stable/swagger/templates/deployment.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "swagger.fullname" . }}
+  labels:
+    app: {{ template "swagger.name" . }}
+    chart: {{ template "swagger.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "swagger.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "swagger.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+          - name: API_URL
+            value: {{ .Values.env.API_URL | quote }}
+          livenessProbe:
+            failureThreshold: 5
+            httpGet:
+              path: /
+              port: {{ .Values.service.internalPort }}
+              scheme: HTTP
+            initialDelaySeconds: 50
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 2
+          readinessProbe:
+            failureThreshold: 10
+            httpGet:
+              path: /
+              port: {{ .Values.service.internalPort }}
+              scheme: HTTP
+            initialDelaySeconds: 50
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 2
+          ports:
+            - containerPort: {{ .Values.service.internalPort }}
```

Comment:
You should probably hard-code the container port. Changing it is likely to break things. Use a named port and reference it by name from service and probes.

Reply:
I removed this setting from the README, but kept them as templated values.

I agree that `Changing it is likely to break things`, however the same can be said for pretty much every other value. If someone chooses to use a different swagger image that exposes a different port, than they can't use this helm chart? Just because the port is hard-coded? that doesn't make sense to me.

Reply:
I entirely disagree with your logic, but it's not worth blocking this chart from being available